
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264581238 -3600
# Node ID 7d9ba85f62eae4ae2b25c5068cbb2c6ed2d20a10
# Parent  17502820c67bea1f869cde31900ad6dd48d03693
libv4l: update upside-down devices list

From: Hans de Goede <hdegoede@redhat.com>

libv4l: update upside-down devices list

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Tue Jan 26 20:35:26 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Wed Jan 27 09:33:58 2010 +0100
@@ -155,6 +155,9 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb071, 0, "ASUSTeK Computer INC.", "N5051Tp",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x04f2, 0xb071, 0, "PEGATRON CORP.", NULL,
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED, 0,
+    "Philco", "S.rie PHN10050" },
   { 0x04f2, 0xb072, 0, "ASUSTeK Computer Inc.        ", "K50IJ     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb106, 0, "ASUSTeK Computer Inc.        ", "N50Vc      ",

